upper-constraints.txt

```diff
@@ -183,7 +183,7 @@ PyYAML===6.0
 beautifulsoup4===4.11.1
 os-net-config===16.0.0
 ovs===2.17.7
-cryptography===38.0.2
+cryptography===42.0.4
```

Comment:
This one is perhaps the most scary

Reply:
I already rebuilt the images based on this change and testing them